apps/server/internal/runtime/docker/docker.go

```
@@ -0,0 +1,429 @@
+
+func (r *Runtime) ensureImage(ctx context.Context, image string) error {
+	if _, err := r.api.ImageInspect(ctx, image); err == nil {
+		return nil
+	} else if !cerrdefs.IsNotFound(err) {
+		return fmt.Errorf("inspect Docker image %q: %w", image, err)
+	}
+	response, err := r.api.ImagePull(ctx, image, client.ImagePullOptions{})
+	if err != nil {
+		return fmt.Errorf("pull Docker image %q: %w", image, err)
+	}
+	defer response.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `response.Close` is not checked

(errcheck)